packages/messageProxy/src/doc/doc.service.ts

```diff
@@ -40,15 +40,15 @@ export class DocService implements OnModuleDestroy {
             } as BroadcastMessage)
         );
 
-        this.actorService.sendChanges(payload);
+        this.actorService.sendChanges(client.id, payload);
```

Comment:
`client.id` is [randomly generated](https://github.com/a-la-fois/a-la-fois/blob/changes/packages/messageProxy/src/ws/ws.gateway.ts#L31-L33)

Do you plan to get it from the client itself?

Reply:
Yeap
When I finish Client Auth `client.id` will contain userId sent by consumer